### DISCUSSION TITLE
Walletconnect modal + Angular, Guide? Example?

### DISCUSSION THREAD

**Discussion by neverlose-lv** [WalletConnect/walletconnect-monorepo#4476] [Developer Support]
Hello. I have not found on the website, how to connect walletconnect modal with an angular project. There is Vue, React and others, but no Angular. Is there any guidance on that? I have searched in internet, but have found only some examples, that uses deprecated dependencies.

Any guide/example on how to use it, which dependencies are required - will be good.
I am interested to have at least two providers: metamask and walletconnect, so the users can login an perform actions not only via pc browser with metamask extension, but also with the phone app, like Trust Wallet.

It would be good if you mension on how to test the thing on a local dev server (e.g. http://locahost:4200/), because walletconnect project settings requirs to enter the project homepage url etc...

**Comment by glitch-txs** [comment]
You can use the JS version in Angular
https://docs.walletconnect.com/web3modal/javascript/about
